king_phisher/plugins.py

```diff
@@ -135,6 +136,15 @@ class OptionString(OptionBase):
 	"""A plugin option which is represented with a string value."""
 	pass
 
+Requirement = collections.namedtuple('Requirement', ('type', 'value', 'comptabile'))
+
+_RequirementMeta = collections.namedtuple('RequirementMeta', ('name', 'title'))
+class _RequirementType(enum.Enum):
```

Comment:
This should be public because you need it in `_compatibility_to_dict` in king_phisher/client/windows/plugin_manager.py.